### PR TITLE
Remove fastload setting and change loading from ugly text to responsive and beatiful icon

### PR DESCRIPTION
**Pull request by ccuser44**
Remove fastload setting and change loading from ugly text to responsive and beatiful icon

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
Nope.

**Comment by ccuser44** [comment]
> Nope.

Why?

**Comment by Dimenpsyonal** [comment]
Because some people might want to continue the regular load
Also for bug reasons

File: MainModule/Client/UI/Default/UserPanel.lua [hunk 1]
```diff
@@ -477,30 +470,39 @@ return function(data, env)
...
-			chatMod = Remote.Get("Setting",{"Prefix","SpecialPrefix","BatchKey","AnyPrefix","DonorCommands","DonorCapes"})
-			settingsData = Remote.Get("AllSettings")
-			Variables.Aliases = playerData.Aliases or {}
-			commandPrefix = chatMod.Prefix
-
-			for _, v in {donorLoad, keyLoad, aliasLoad, clientLoad, gameLoad} do
-				v:Destroy()
+			while loadingIcons[1].Parent do
+				for _, v in loadingIcons do
+					v.Rotation = -(os.clock() - start)/(1/60)*10
```
[the remark this answers is not in the document]
Comment (ccuser44) [reply MainModule/Client/UI/Default/UserPanel.lua:492]: Thx \:)

**Comment by ccuser44** [comment]
> Because some people might want to continue the regular load

But I don't see any good reason for people wan't to continue it. It functions the same as before expect it's faster and less buggy.

**Comment by Dimenpsyonal** [comment]
Some people might want to not use the loading icon and instead just want the UI

**Comment by ccuser44** [comment]
> Some people might want to not use the loading icon and instead just want the UI

But, why?

**Comment by Dimenpsyonal** [comment]
So hard-loading can still be available

**Review by Dimenpsyonal — APPROVED** [review]

**Pull request merged**